### PR TITLE
Update documentaition: retain-old-md, compatibility, koji simple, merge methods

### PR DESCRIPTION
**Pull request by kontura**
The first commit addresses https://github.com/rpm-software-management/createrepo_c/issues/79.

### CODE REVIEW

File: doc/createrepo_c.8 [hunk 1]
```diff
@@ -104,7 +104,10 @@ Include the file\(aqs checksum in the metadata filename, helps HTTP caching (def
 Do not include the file\(aqs checksum in the metadata filename.
 .SS \-\-retain\-old\-md NUM
 .sp
-Keep around the latest (by timestamp) N copies of the old repodata.
+Specify NUM to 0 to remove all repodata present in old repomd.xml or any other positive number to keep all old repodata.
+.sp
+Use --compatibily flag to get the behavior of original createrepo:
```
Comment (Conan-Kudo) [inline doc/createrepo_c.8]: Isn't this `--compatibility`?

File: doc/mergerepo_c.8 [hunk 2]
```diff
@@ -62,7 +62,24 @@ Do not merge updateinfo metadata
...
+Available merge methods:
+.P
+.B repo
+.RS 3
+(default) select package from the first specified repository.
+.RE
+.sp
+.B nvr
+.RS 3
+select pacakge with the higher version and release.
```
Comment (Conan-Kudo) [inline doc/mergerepo_c.8]: `pacakge` -> `package`

File: src/cmd_parser.c [hunk 3]
```diff
@@ -133,7 +133,9 @@ static GOptionEntry cmd_entries[] =
     { "simple-md-filenames", 0, 0, G_OPTION_ARG_NONE, &(_cmd_options.simple_md_filenames),
       "Do not include the file's checksum in the metadata filename.", NULL },
     { "retain-old-md", 0, 0, G_OPTION_ARG_INT, &(_cmd_options.retain_old),
-      "Keep around the latest (by timestamp) N copies of the old repodata.", "NUM" },
+      "Specify NUM to 0 to remove all repodata present in old repomd.xml or any other positive number to keep all old repodata. "
+      "Use --compatibily flag to get the behavior of original createrepo: "
```
Comment (Conan-Kudo) [inline src/cmd_parser.c]: `--compatibily` -> `--compatibility`

Comment (kontura) [reply src/cmd_parser.c]: Thank you! I have updated the PR, apparently I can't write..

**Pull request merged**